tensorflow/lite/testing/op_tests/conv.py

```diff
@@ -39,6 +39,20 @@ def make_conv_tests(options):
           "constant_filter": [True, False],
           "channel_multiplier": [1, 2],
           "fully_quantize": [False],
+          "quantize_mode_16x8": [False],
+          "dynamic_range_quantize": [False]
+      },
+      {
+          "input_shape": [[1, 3, 4, 3]],
+          "filter_shape": [[1, 1], [2, 3]],
+          "strides": [[1, 1, 1, 1]],
+          "dilations": [[1, 1, 1, 1]],
+          "padding": ["SAME"],
+          "data_format": ["NHWC"],  # TODO(aselle): NCHW  would be good
+          "constant_filter": [True, False],
+          "channel_multiplier": [1, 2],
+          "fully_quantize": [False],
```

Comment:
Is fully quantize supposed to be True here? In order for teh quantize_mode_16x8 flag to be run.